lib/WP_Auth0_LoginManager.php

```diff
@@ -336,12 +331,17 @@ public function implicit_login() {
     }
   }
 
-  // Does all actions required to log the user in to wordpress, invoking hooks as necessary
-  // $user (stdClass): the WP user object, such as returned by get_user_by(...)
-  // $user_profile (stdClass): the Auth0 profile of the user
-  // $is_new (boolean): `true` if the user was created on Wordress, `false` if not.  Don't get confused with Auth0 registrations, this flag will tell you if a new user was created on the WordPress database.
-  // $id_token (string): the user's JWT
-  // $access_token (string): the user's access token.  It is not provided when using the **Implicit flow**.
+  /**
+   * Does all actions required to log the user in to wordpress, invoking hooks as necessary
+   *
+   * @param object $user - the WP user object, such as returned by get_user_by()
+   * @param object $userinfo - the Auth0 profile of the user
+   * @param bool $is_new - `true` if the user was created in the WordPress database, `false` if not
+   * @param string $id_token - user's ID token returned from Auth0
+   * @param string $access_token - user's access token returned from Auth0; not provided when using implicit_login()
```

Comment:
AFAIK the AcT is always returned. The one that might be missing is the IdT.

Reply:
That's true ... but the `$id_token` param was already there

Reply:
So why isn't the access token provided "when using `implicit_login()`"?